priv/migrations/20230509120001_index_oracle_queries_responses.ex

```diff
@@ -0,0 +1,51 @@
+defmodule AeMdw.Migrations.IndexOracleQueriesResponses do
+  @moduledoc """
+  Indexes new Model.oracle_query response_txi_idx attribute.
+  """
+
+  alias :aeser_api_encoder, as: Enc
+  alias AeMdw.Db.WriteMutation
+  alias AeMdw.Db.Model
+  alias AeMdw.Db.State
+  alias AeMdw.Db.Util, as: DbUtil
+
+  require Model
+
+  @spec run(State.t(), boolean()) :: {:ok, non_neg_integer()}
+  def run(state, _from_start?) do
+    mutations =
+      state
+      |> DbUtil.transactions_of_type(:oracle_response_tx, :forward, nil, nil)
+      |> Stream.map(fn response_txi_idx ->
+        oracle_response_tx = DbUtil.read_node_tx(state, response_txi_idx)
+        oracle_pk = :aeo_response_tx.oracle_pubkey(oracle_response_tx)
+        query_id = :aeo_response_tx.query_id(oracle_response_tx)
+
+        case State.get(state, Model.OracleQuery, {oracle_pk, query_id}) do
+          {:ok, {:oracle_query, index, txi_idx}} ->
+            oracle_query =
+              Model.oracle_query(
+                index: index,
+                txi_idx: txi_idx,
+                response_txi_idx: response_txi_idx
+              )
+
+            WriteMutation.new(Model.OracleQuery, oracle_query)
+
+          :not_found ->
+            IO.puts("""
```

Comment:
Could we log these so we can have a record of them on staging? When someone asks we could search for them on the log file.

Reply:
These responses have already been logged here https://github.com/aeternity/ae_mdw/blob/master/lib/ae_mdw/db/mutations/oracle_response_mutation.ex#L66 and can be found on the logs. This is just to inform during migrating